src/ui/components/VPNPanel.qml

```diff
@@ -85,6 +85,12 @@ ColumnLayout {
             Layout.topMargin: 12
             Layout.fillWidth: true
             wrapMode: Text.WordWrap
+            onLineLaidOut: {
+                if (text.indexOf(" ") > 0)
```

Comment:
```
if (text.indexOf(" ") == -1 && line.implicitWidth > panel.width) {
   wrapMode = Text.WrapAnywhere
}
```

Reply:
Updated! I ended up just removing all of this in favor of `WrapWordAtBoundaryOrAnywhere` which does the same thing.